gravitee-apim-console-webui/src/management/application/creation/components/application-creation-form.component.html

```diff
@@ -52,13 +52,19 @@
 
     <mat-form-field>
       <mat-label>Select Groups</mat-label>
-      <mat-hint>Select user groups to add to the application</mat-hint>
       <mat-select formControlName="groups" multiple>
         @for (group of groupsList | async; track group) {
           <mat-option [value]="group.id">{{ group.name }}</mat-option>
         }
       </mat-select>
+      <mat-hint>Select user groups to add to the application</mat-hint>
     </mat-form-field>
+    @if (!hasGroupsToAdd && requireUserGroups) {
+      <mat-hint
```

Comment:
I think that there is something wrong here with the display
<img width="1728" alt="Capture d’écran 2025-04-02 à 12 39 52" src="https://github.com/user-attachments/assets/dfd487a6-e198-4154-9cf3-edef5f09a418" />

Reply:
I would have not use error here but more info or warning colors. Wdyt ? 
Otherwise yes it's better.